mxnet/inference/docker/1.9/py3/Dockerfile.cpu

```diff
@@ -79,6 +83,20 @@ RUN apt-get update \
 
 WORKDIR /
 
+RUN curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases/download/${MAMBA_VERSION}/Mambaforge-${MAMBA_VERSION}-Linux-x86_64.sh \
```

Comment:
When we pull the container and type `conda` it fails to work. This is leading to a few test failures. 

Can we fix this? We can refer to other dockerfiles that have conda setup, for eg. https://github.com/aws/deep-learning-containers/blob/master/pytorch/training/docker/1.12/py3/cu113/Dockerfile.gpu#L159-L186

Reply:
Added /opt/conda/bin to path. Should be resolved.